### DISCUSSION TITLE
Disable automatic messages when changing preset

### DISCUSSION THREAD

**Discussion by phillix2** [shanteacontrols/OpenDeck#231] [Q&A]
Can I disable the sending of all the controller's messages when switching from one preset to another?

I'm trying to control different parameteers with the same potentiometers by changing the presets, but the last status from my initial preset is sent to the controllers of the second when I change them.

**Comment by paradajz** [comment]
No, you can't disable it as the setting isn't implemented.

**Reply by phillix2** [reply]
Tks for answering.

**Comment by paradajz** [comment]
This is now implemented on master branch.
